Sources/ProjectDescription/Plugins.swift

```diff
@@ -0,0 +1,20 @@
+import Foundation
+
+/// A `Plugin` manifest allows for defining extensions for Tuist.
+///
+/// Supported plugins include:
+/// - ProjectDescriptionHelpers
+///     - These are plugins designed to be usable by any other manifest excluding `Config` and `Plugin`.
+///     - The source files for these helpers must live under a ProjectDescriptionHelpers directory in the location where `Plugin` manifest lives.
+///
+public struct Plugin: Codable, Equatable {
```

Comment:
Nitpick: I'd rename the file to `Plugin.swift`, so the names correspond to each other

Reply:
Oops nice catch